notebooks/MAST/m_dwarf_tess/m_dwarf_flares_tess.ipynb

```diff
@@ -0,0 +1,1285 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    sigma_2 = convolve(np.square(cleaned_twice - running_mean), g)
</pre><p>This 'g' factor does not appear in the notebook, so this cell failed when I ran it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>